src/presentation/http/router/note.test.ts

```diff
@@ -359,7 +412,7 @@ describe('Note API', () => {
         },
         url: `/note`,
         body: {
-          parentId: 'Hu8Gsm0sA1',
+          parentId: 'ishvm5qH84',
```

Comment:
Is there any way to eliminate the magic data?

Reply:
changed